Mail/Template.php

```diff
@@ -249,7 +278,18 @@ public function getProcessedTemplate(array $variables = [])
         } catch (\Exception $e) {
             $this->cancelDesignConfig();
             throw new \LogicException(__($e->getMessage()), $e);
+        } finally {
+            /**
+             * After Magento v2.3.4 need set strict mode
+             *
+             * @customization START
+             */
+            if (version_compare($this->sailthruSettings->getMagentoVersion(), '2.3.4', '>=')) {
```

Comment:
same comment as above: Should the comment read `In Magento v2.3.4 and later, need set strict mode`?